src/main/java/nl/hsac/fitnesse/fixture/fit/MapColumnFixture.java

```diff
@@ -376,6 +395,26 @@ public void doCell(Fixture aFixture, Parse aCell) {
 
     }
 
+    /**
+     * Functionality to compare returning cell value with a dynamic (with symbols) text.
+     * See also HsacExamples.FitTests.ArraysAndSymbolsComparison#Compare result with string containing a symbol
+     * @param originalCellText, ie randomText$[symbol1]MoreText where symbol1=ValueOfSymbol1
+     * @return originalCellText with the symbols resolved to values, ie: randomTextValueOfSymbol1MoreText
+     *      or null if the originalCellText doesn't contains text with symbols
+     * @throws NoSuchSymbolException if the used symbol(s) don't exist
+     */
+    private String resolveStringWithSymbols(String originalCellText) throws NoSuchSymbolException {
+        Pattern p = Pattern.compile(REGEX_STRING_WITH_SYMBOL);

```

Comment:
Why don't you store the pattern in a final static (instead of only the String and compiling each time)?